chef/data_bags/crowbar/migrate/nova/032_add_zvm_attributes.rb

```diff
@@ -0,0 +1,16 @@
+def upgrade ta, td, a, d
+  a['zvm'] = ta['zvm']
+  d['element_states'] = td['element_states']
+  d['element_order'] = td['element_order']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.